inst/extdata/installPackages.R

```diff
@@ -21,7 +45,9 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
     },
     warning = function(w) {
       # if package p is not installed...
-      if (any(grepl("(DESCRIPTION file of package .+ is missing or broken|no package .+ was found)", w))) {
+      if (any(grepl(
```

Comment:
gsplot is installed on line 115, and I have a very tenuous understanding of gsplot's repositories. That line is really just copy-and-pasted from Zack's Slack recommendation.